### PR TITLE
[regression test]Optimize waitForSchemeChangeDone

### PR DESCRIPTION
**Pull request by cjj2010**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by cjj2010** [comment]
run buildall

File: regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/Suite.groovy [hunk 1]
```diff
@@ -522,8 +523,16 @@ class Suite implements GroovyInterceptable {
...
     }
 
-    void waitForSchemaChangeDone(Closure actionSupplier) {
+    void waitForSchemaChangeDone(Closure actionSupplier, String insertSql = null, boolean cleanOperator = false,String tbName=null) {
         runAction(new WaitForAction(context), actionSupplier)
+        if (ObjectUtils.isNotEmpty(insertSql)){
+            sql insertSql
+        }
+        if (cleanOperator==true){
+            if (ObjectUtils.isEmpty(tbName)) throw new RuntimeException("tbName cloud not be null")
```
[the remark this answers is not in the document]
Comment (cjj2010) [reply regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/Suite.groovy:532]: Good idea, I'll make some improvements

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Review by Yukang-Lian — APPROVED** [review]

**Pull request merged**